packages/token-lists/package.json

```diff
@@ -29,7 +29,8 @@
     "@uniswap/token-lists": "^1.0.0-beta.21",
     "ajv": "6.12.2",
     "graphql": "^15.5.0",
-    "graphql-request": "^3.4.0"
+    "graphql-request": "^3.4.0",
+    "slugify": "^1.6.0"
```

Comment:
Since this is used in a generation script lets put this in `devDependencies`

Reply:
Fixed in 87dc48e